test/index.js

```diff
@@ -1,34 +1,38 @@
 "use strict"
-var test = require("tap").test
-var requireInject = require("require-inject")
+const tap = require('tap');
+const requireInject = require('require-inject');
```

Comment:
Quote type changed here and nowhere else. (My modern personal style is single quotes everywhere, so I'm not opposed, but consistency is more important.)